src/main/java/game/baseball/player/HumanPlayer.java

```diff
@@ -0,0 +1,25 @@
+package game.baseball.player;
+
+import game.baseball.ball.Balls;
+import game.baseball.util.number.NumberGenerator;
+
+import static game.baseball.util.setting.GameSetting.GAME_SETTING;
+
+public class HumanPlayer implements Player {
+
+    private Balls balls;
+    private final NumberGenerator numberGenerator;
```

Comment:
```suggestion
    private final NumberGenerator numberGenerator;
    private Balls balls;
```

ComputerPlayer의 내용과 같습니다.